config/config.go

```diff
@@ -340,8 +339,6 @@ type (
 		SQLITE3 SQLITE3 `yaml:"SQLITE3"`
 		// SplitDBSize is the config for DB's split file size
 		SplitDBSizeMB uint64 `yaml:"splitDBSizeMB"`
-		// SplitDBHeight is the config for DB's split start height
-		SplitDBHeight uint64 `yaml:"splitDBHeight"`
```

Comment:
good to start with one option. I personally prefer "SplitDBHeight" to "SplitDBSizeMB"